gnssanalysis/gn_io/igslog.py

```diff
@@ -109,21 +134,25 @@ def parse_igs_log(filename_array: _np.ndarray) -> _np.ndarray:
     with open(file_path, "rb") as file:
         data = file.read()
```

Comment:
Also as you like to comment on everyone else PR. 
the time you change this file, could you review and update the documentation `on-line sphinx style` with input/output descriptions ;)

Reply:
I've gone through and updated the doc-strings for all functions in this file